projects/ngrx.io/content/marketing/events.json

```
@@ -0,0 +1,113 @@
+[
+  {
+    "name": "NgColombia",
+    "url": "http://www.ngcolombia.com",
+    "location": "Medellín, Colombia",
+    "startDate": "09-06-2018",
+    "endDate": "09-07-2018"
+  },
+  {
+    "name": "DevFestATL",
+    "url": "http://devfestatl.com",
+    "location": "Atlanta, Georgia",
+    "endDate": "09-22-2018"
+  },
+  {
+    "name": "Framework Summit",
+    "url": "http://frameworksummit.com",
+    "location": "Park City, Utah",
+    "endDate": "09-22-2018"
+  },
+  {
+    "name": "DevFestATL",
+    "url": "http://devfestatl.com",
+    "location": "Atlanta, Georgia",
+    "startDate": "10-02-2018",
+    "endDate": "10-03-2018"
+  },
+  {
+    "name": "AngularMix",
+    "url": "http://www.angularmix.com",
+    "location": "Orlando, Florida",
+    "startDate": "10-10-2018",
+    "endDate": "10-12-2018"
+  },
+  {
+    "name": "AngularConnect",
+    "url": "http://www.angularconnect.com",
```

```suggestion
    "url": "https://www.angularconnect.com",
```

The non HTTPs entry doesn't seem to work

---

updated in fc30ba5